cl_sii/dte/data_models.py

```diff
@@ -529,21 +698,29 @@ class DteXmlData(DteDataL1):
         and :func:`cl_sii.libs.crypto_utils.x509_cert_der_to_pem`.
     """
 
-    emisor_giro: Optional[str] = dc_field(default=None)
+    emisor_giro: Optional[str] = None
     """
     "Giro" of the "emisor" of the DTE.
     """
 
-    emisor_email: Optional[str] = dc_field(default=None)
+    emisor_email: Optional[str] = None
     """
     Email address of the "emisor" of the DTE.
     """
 
-    receptor_email: Optional[str] = dc_field(default=None)
+    receptor_email: Optional[str] = None
     """
     Email address of the "receptor" of the DTE.
     """
 
+    referencias: Optional[Sequence[DteXmlReferencia]] = None
+    """
+    List of structs for ``Referencia`` XML elements.
+
+    ..warning::
+        The items MUST be ordered according to their ``numero_linea_ref``.
+    """
+
     def __post_init__(self) -> None:
```

Comment:
If `DteXmlData` is going to be converted into Pydantic Dataclass, then either move the validations performed in `__post_init__` to `__post_init_post_parse__`, or refactor them as Pydantic validators. Either way, please do it in a separate commit and/or pull request.

Pydantic's documentation says:

> When substituting usage of `dataclasses.dataclass` with `pydantic.dataclasses.dataclass`, it is recommended to move the code executed in the `__post_init__` method to the `__post_init_post_parse__` method, and only leave behind part of code which needs to be executed before validation.

Source: https://pydantic-docs.helpmanual.io/usage/dataclasses/#difference-with-stdlib-dataclasses